frontend/cli/cmd_replay.go

```diff
@@ -97,6 +98,8 @@ func (c *replayCmd) Run(ctx context.Context, client ftlv1connect.VerbServiceClie
 	}
 	requestJSON := events.Msg.GetEvents()[0].GetCall().Request
 
-	logger.Infof("Calling %s with body:\n%s", c.Verb, requestJSON)
+	logger.Infof("Calling %s with body:", c.Verb)
```

Comment:
BTW just a not on levels - we're preferring to put everything "non-essential" at debug level. We used to use info for everything, but users found it confusing and overwhelming.